CHANGES.txt

```diff
@@ -1,3 +1,5 @@
+5.0
+ * Standaridize storage configuration parameters' names. Support unit suffixes. (CASSANDRA-15234)
```

Comment:
```suggestion
 * Standardize storage configuration parameters' names. Support unit suffixes. (CASSANDRA-15234)
```

I'll again leave the 4.1 vs. 5.0 discussion for the Jira...

Reply:
Sure, I just left it for now as is. This will be the last thing to change pre-commit :-)